source/tutorial.rst

```diff
@@ -271,18 +271,45 @@ Code Tracking
 
 On a statement level you can choose during code tracking to animate stepping into statements or stepping over them using the ``stepInto`` and ``stepOver`` blocks.
 
-.. code:: javascript
+.. code:: kotlin
     
     ...
-    stepInto {
+    @stepInto {
     let x = f(y);       // This will animate the execution of statements inside the function
     }
 
-    stepOver {
+    @stepOver {
     let z = f(y);       // This will simply step over the statement
     }
     ...
 
+Subtitles
+^^^^^^^^^^^^^^
+
+A subtitle annotation will allow you to add descriptive text to you animation. There are two types of subtitles:
```

Comment:
Really minor but present tense here ("a subtitle annotation allows you to...") might be more consistent with the rest of the doc? Everything looks good though